app/models/technical_metadata.rb

```diff
@@ -0,0 +1,8 @@
+class TechnicalMetadata < ActiveFedora::Base
+  include Hydra::Works::WorkBehavior
```

Comment:
@csyversen and @carrickr can you explain the inclusion of `Hydra::Works::WorkBehavior` here? I can't think of why `TechnicalMetadata` class should behave like a `Hydra::Works::Work`. The technical metadata that comes from `hydra-works` is located on `Hydra::PCDM::File` objects. I'm having a hard time conceptualizing the relationships you have established between `FileSet` and `TechnicalMetadata` objects (See https://github.com/samvera/hydra-works/pull/292 for implementation details)

Reply:
we had to include `Hydra::Works::WorkBehavior` because we couldn't extend the schema/properties of the fileset itself due to upstream bugs and we couldn't declare the class as a `PCDM::File` object because there was no file associated with it (we tried it a few weeks ago and kept getting errors)

Reply:
Do you have any more information about these upstream bugs? I don't think the approach is correct, the `Image` is the works that FileSets are associated with and I think the relationship between `FileSets` and `TechnicalMetadata` doesn't make sense on a conceptual level.

Reply:
i'd have to go back and look in old commits to see if i can replicate the exact error message, but trying to extend the `FileSet` with `apply_schema Schemas::<Whatever>, Schemas::GeneratedResourceSchemaStrategy.new` wasn't successful. Even trying to override the FileSet model and trying to add in properties directly onto it caused errors. We might want to talk to MBK about this tomorrow, I went to him with these problems/error messages and he asked in the dev channel and people didn't know there either. 

And I know what we have doesn't fit into the PCDM model (since TechnicalMetadata isn't defined as a library-capital-w Work, but conceptually this implementation is similar to a regular ActiveRecord `has_one`/ `belongs_to` relationship), but it does get the technical metadata safely into fedora, associate the files with their tech md, allows the staff to continue with their work, and was able to be implemented before our go live date. We tried to follow a standard PCDM style model but were met with showstopper-style bugs, so we had to get creative.

Reply:
(and fwiw I did try to implement the TechnicalMetadata model as a straight activefedora object and not include any hydra-works models/behaviors too because i thought they weren't applicable either, but that also errored out)